sdk/core/azure-core-tracing-opencensus/azure/core/tracing/ext/opencensus_span/__init__.py

```diff
@@ -215,61 +206,67 @@ def get_trace_parent(self):
         return self.to_header()["traceparent"]
 
     @classmethod
-    def link(cls, traceparent, attributes=None):
-        # type: (str, Attributes) -> None
-        """
-        Links the context to the current tracer.
+    def link(cls, traceparent: str, attributes: Attributes = None) -> None:
+        """Links the context to the current tracer.
 
         :param traceparent: A complete traceparent
         :type traceparent: str
+        :param attributes: Attributes to be added to the link
+        :type attributes: dict or None
         """
         cls.link_from_headers({"traceparent": traceparent}, attributes)
 
     @classmethod
-    def link_from_headers(cls, headers, attributes=None):
-        # type: (Dict[str, str], Attributes) -> None
-        """
-        Given a dictionary, extracts the context and links the context to the current tracer.
+    def link_from_headers(cls, headers: Dict[str, str], attributes: Attributes = None) -> None:
```

Comment:
minor nit - I know `Attributes` is defined as Optional[Dict]] but I think it would just look nicer while reading `attributes: Optional[Attributes] = None` across other signatures

Reply:
Yea, I agree it would look nicer 👍 . I pushed a commit to move the Optional to the method signatures when Attributes is used (in both OTel and OpenCensus plugins).